src/lucky/renderable.cr

```diff
@@ -69,29 +69,43 @@ module Lucky::Renderable
 
   # Render an HTMLPage with a status other than 200
   #
+  # The status can either be a Number, a HTTP::Status, or a Symbol that corresponds to the HTTP::Status.
+  #
   # ```
   # class SecretAgents::Index < BrowserAction
   #   get "/shhhh" do
   #     html_with_status IndexPage, 472, message: "This page can only be seen with special goggles"
   #   end
   # end
   # ```
+  # See Crystal's
+  # [HTTP::Status](https://crystal-lang.org/api/latest/HTTP/Status.html)
+  # enum for more available http status codes.
   macro html_with_status(page_class, status, **assigns)
-    {% if !status.is_a?(NumberLiteral) %}
+    {% if status.is_a?(NumberLiteral) %}
+      html {{ page_class }}, _with_status_code: {{ status }}, {{ **assigns }}
+    {% elsif status.is_a?(SymbolLiteral) %}
+      html {{ page_class }}, _with_status_code: HTTP::Status::{{ status.upcase.id }}.value, {{ **assigns }}
```

Comment:
Nice! I didn't know you could call `upcase` right on the symol at the macro level like that.

Reply:
It was also a first for me